server/src/api/context.go

```diff
@@ -172,6 +172,20 @@ func (s *Server) BoardAuthenticatedContext(next http.Handler) http.Handler {
 	})
 }
 
+func (s *Server) AnonymousLoginEnabledContext(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		log := logger.FromRequest(r)
+
+		if !s.anonymousLoginEnabled {
+			log.Errorw("not allowed to login anonymously")
+			common.Throw(w, r, common.ForbiddenError(errors.New("not authorized to login anonymously")))
```

Comment:
404 seems more fitting

Reply:
how so? the route still exists but musn't be used.